src/core/zowe/core_for_zowe_sdk/validators.py

```diff
@@ -31,10 +33,22 @@ def validate_config_json(path_config_json: str, path_schema_json: str):
         Provides details if config.json doesn't match schema.json, otherwise it returns None.
     """
 
+    # checks if the path_schema_json point to an internet URI and download the schema using the URI
+    if path_schema_json[:8] in ("https://", "http://"):
```

Comment:
Grabbing the first 8 characters here won't always work. `https://` is 8 characters long but `http://` is only 7.

Reply:
When you write unit tests for this method, please include one that validates JSON using an online schema like the following: https://json-schema.org/draft/2020-12/schema